### PR TITLE
Add supporting link for final statement

### PR DESCRIPTION
**Pull request by JeremyTBradshaw**
The last statement on the page says to make sure to prepare the users for the changes they can expect, but there's nothing on this page that describes what to expect.  I added a sentence and link to where that info can be seen.

### CODE REVIEW

File: microsoft-365/compliance/enable-autoexpanding-archiving.md [hunk 1]
```diff
@@ -137,4 +137,4 @@ Keep the following things in mind after you enable auto-expanding archiving:
 
 ## Next steps
 
-Make sure you prepare end users for the changes they can expect after enabling their mailbox for auto-expanding archives.
+Make sure you prepare end users for the changes they can expect after enabling their mailbox for auto-expanding archives.  For a brief overview of the client experience, see the "What gets moved to the additional archive storage space?" section in [Learn about auto-expanding archiving](https://learn.microsoft.com/en-us/microsoft-365/compliance/autoexpanding-archiving?view=o365-worldwide#what-gets-moved-to-the-additional-archive-storage-space).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/enable-autoexpanding-archiving.md]:
```suggestion
Make sure you prepare end users for the changes they can expect after enabling their mailbox for auto-expanding archives.  For a brief overview of the client experience, see [What gets moved to the additional archive storage space?](https://learn.microsoft.com/microsoft-365/compliance/autoexpanding-archiving#what-gets-moved-to-the-additional-archive-storage-space).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JeremyTBradshaw, kindly commit the suggested change. Many thanks.

File: microsoft-365/compliance/enable-autoexpanding-archiving.md [hunk 2]
```diff
@@ -137,4 +137,4 @@ Keep the following things in mind after you enable auto-expanding archiving:
 
 ## Next steps
 
-Make sure you prepare end users for the changes they can expect after enabling their mailbox for auto-expanding archives.
+Make sure you prepare end users for the changes they can expect after enabling their mailbox for auto-expanding archives.  For a brief overview of the client experience, see [What gets moved to the additional archive storage space?](https://learn.microsoft.com/microsoft-365/compliance/autoexpanding-archiving#what-gets-moved-to-the-additional-archive-storage-space).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/enable-autoexpanding-archiving.md]:
```suggestion
Make sure you prepare end users for the changes they can expect after enabling their mailbox for auto-expanding archives. For a brief overview of the client experience, see [What gets moved to the additional archive storage space?](/microsoft-365/compliance/autoexpanding-archiving#what-gets-moved-to-the-additional-archive-storage-space).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JeremyTBradshaw, kindly commit the suggested change to remove the extra space and change the link to a relative link. Many thanks.

**Comment by JeremyTBradshaw** [comment]
Done, thanks again.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @JeremyTBradshaw.

**Pull request closed without merging**